transformation/v2-to-fhirr4-service/README.md

```diff
@@ -29,6 +29,34 @@ You do not have to write code from scratch but reuse these existing services whe
     --data-raw 'MSH|^~\&|EPIC|EPICADT|SMS|SMSADT|202211031408|CHARRIS|ADT^A01|1817457|D|2.8'
     ```
 
+## Customization
+
+You can customize the v2 segments to FHIR transformation logic by implementing a service with custom mappings and configure the v2-to-fhirr4-service to invoke the custom service. This custom service can be implemented from Ballerina by using the template or use the OAS definition provided(resources/custom-mapping-service-openapi.yaml) to implement your service using any other technology. Following is the command to initiate the Ballerina template. 
+
+```bash
+bal new <service_name> -t ballerinax/health.fhir.templates.r4.v2tofhircustomsvc
+```
+
+You can configure the v2-to-fhir-service to point your custom service to invoke the mapping endpoints. Following is a sample configuration structure to invoke a custom service.
+
+```toml
+[serviceConfig]
+baseUrl = "http://localhost:9091/v2tofhir"
+
+[serviceConfig.segmentToAPI]
+NK1 = "/segment/nk1"
+PID = "/segment/pid"
+```
```

Comment:
What's the file name of this config? Is it Config.toml? Let's mention it.

Reply:
yes. fixed